### PR TITLE
Ensure test prep creates a main camera in the scene

### PR DESCRIPTION
**Pull request by davidkline-ms**
Previously, the play mode tests relied upon behavior in MRTK where it creates a main camera if one cannot be found in the scene.

In preparation of MRTK enforcing correct scene setup by the developer (in 2.3.0), test preparation now ensures a main camera is present in the scene before adding the MRTK object.

### CODE REVIEW

File: Assets/MixedRealityToolkit.Tests/TestUtilities.cs [hunk 1]
```diff
@@ -135,8 +135,18 @@ public static void InitializeMixedRealityToolkitAndCreateScenes(bool useDefaultP
             InitializeMixedRealityToolkit(useDefaultProfile);
         }
 
+        public static void InitializeCamera()
+        {
+            if (Camera.main == null)
+            {
+                new GameObject("Main Camera", typeof(Camera), typeof(AudioListener)) { tag = "MainCamera" }.GetComponent<Camera>();
```
Comment (julenka) [inline Assets/MixedRealityToolkit.Tests/TestUtilities.cs]: For my own education, why did you need the .GetComponent<Camera>() at the end there?

Comment (keveleigh) [reply Assets/MixedRealityToolkit.Tests/TestUtilities.cs]: Ooh...looks like more was copied over from the `CameraCache` script than necessary. Doesn't do anything in this case and could be removed
